vmdb/app/controllers/ems_common.rb

```diff
@@ -336,17 +340,16 @@ def update_button_validate
     set_record_vars(verify_ems, :validate)
     @in_a_form = true
     @changed = session[:changed]
-    begin
-      result = verify_ems.verify_credentials(params[:type])
-    rescue StandardError => bang
-      add_flash("#{bang}", :error)
+
+    # validate button should say "revalidate" if the form is unchanged
```

Comment:
Note, I still don't know what to do with Host's update form validate button since that catches a ssh key mismatch exception.  Maybe I'll do that after getting this issue fixed since this is really a problem with the Ems having a bad auth status and the validate button doesn't do anything.